src/main/java/es/tid/fiware/iot/ac/pap/PoliciesEndpoint.java

```diff
@@ -41,23 +44,33 @@
 
     private PolicyDao dao;
 
-    public PoliciesEndpoint(PolicyDao dao) {
+    private MetricRegistry metrics;
+
+    public PoliciesEndpoint(PolicyDao dao, MetricRegistry metrics) {
         this.dao = dao;
+        this.metrics = metrics;
     }
 
     @GET
     @UnitOfWork
+    @Correlator
+    @Timed
     public Response getPolicy(@Tenant String tenant,
                               @Correlator String correlator,
                               @PathParam("subject") String subject,
                               @PathParam("policyId") String policyId) {
-        
+
         LOGGER.debug("Getting policy with id [{}] for [{}] and subject [{}]", policyId, tenant, subject);
-        
+
         String id = URLEncoding.decode(policyId);
 
         Policy p = dao.loadPolicy(tenant, subject, id);
+        metrics.counter("incomingTransactionRequestSize").inc(tenant.length() +
```

Comment:
This sum does not really give us the Request Size, don't you think? It gives us the size of the interesting contents (I mean, the values inside the target XML elements) but it leaves away all the XML boilerplate used to codify it. Maybe I'm wrong and that's not the case, but if I'm not, this should be noted in the documentation, as the main reason to have this metrics is bandwidth analysis, and it won't be much useful considering those small bits.